homework/week-0/src/nyc/c4q/JVidals1/DeckofCards.txt

```diff
@@ -0,0 +1,83 @@
+package nyc.c4q.JVidals1;
+import javax.smartcardio.Card;
+
+/**
+ * Created by s3a on 5/1/15.
+ * Janneisy Vidals
+ * 2.1 Accesscode
+ * HW Excercises
+ * Description:
+ * Due March 27, 2015.
+ */
+
+
+public abstract class DeckofCards
+{
+    public static final int NumOfCards = 52;
+
+
+    private Card[] deckofCards; // has  all 52 cards ..getpieces
+    private int    currentCard; // deal this card in deck
+
+    //Create a contructor method to make 52 cards in a deck.
+    public DeckofCards()
+    {
+
+        deckofCards = new Card[NumOfCards]; // Creating an array
+
+        int i = 0; // intialize at 0;
+
+        for(int suit = Card.DIAMONDS; suit <= Card.SPADES; suit++)
+        {
+            for(int rank = 1; rank <= 13; rank++) {
+                deckofCards[i++] = new Card (suit, rank);
+
+                 // putting card in position i
+            }
+            currentCard = 0; //Fresh deck of cards
+        }
+    }
+    // create a method that shuffles(n) : shuffle the deck n exchanges
```

Comment:
I get what this does - maybe we can call it `swap` and have a `shuffle` method call this one? `shuffle` sounds like shuffling the entire deck instead of swapping two cards.